test/float/set_rm_invalid_fail.c

```diff
@@ -0,0 +1,14 @@
+#include "smack.h"
+#include <fenv.h>
+
+// @expect error
+
+int main(void)
+{
```

Comment:
Move this into the previous line, like this: `int main(void) {`
Do that with all your regressions to make them consistent with others.